.github/workflows/cicd.yml

```diff
@@ -1,12 +1,33 @@
 name: Continuous Testing and Publication from 'dev'
 
-on: [push]
+on: [push] # yamllint disable-line rule:truthy
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
+  run-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Checkout actions repository
+        uses: actions/checkout@v2
+        with:
+          repository: Exabyte-io/actions
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          ref: 4d3e0fc415f6eb28739a331e7f44cd7ba242668f
```

Comment:
Can be omitted, actions PR is merged now

Reply:
done